### DISCUSSION TITLE
Not possible to have different effect at same effect strength?

### DISCUSSION THREAD

**Discussion by Foxtrot39** [FakeFishGames/Barotrauma#12077] [Code questions and modding support]
So I have this code that used to work  

  Effect minstrength="0" maxstrength="600" strengthchange="-0.5" resistancefor="radiationsickness" minresistance="0.8" maxresistance="0.95" 

Effect minstrength="0" maxstrength="600" strengthchange="-0.5" resistancefor="contamination" minresistance="1" maxresistance="1" 

But now I get a effect strength overlap error saying one is no longer working (and from testing it is really not working anymore)

How do I cram both different resistances into the same effect zone then?

**Comment by BiffW** [comment]
> How do I cram both different resistances into the same effect zone then?

By simply separate the effects into 2 kinds of afflictions

**Comment by Foxtrot39** [comment]
Which I woud llike to avoid since it make me rework multiple afflictions that check it or remove it depending on circumstances and the code is already lag inducing as is
